lnrpc/routerrpc/router_server.go

```diff
@@ -1192,11 +1195,11 @@ func getPair(amtMsat lnwire.MilliSatoshi, amtSat btcutil.Amount,
 	case timeSet && amountSet:
 		return amt, time.Unix(timestamp, 0), nil
 
-	case timeSet && !amountSet:
+	case timeSet && (!amountSet && isFailure):
```

Comment:
```suggestion
	case timeSet && !amountSet && isFailure:
```
nit: would be nice to add some doc above each `case`.

Reply:
fixed syntax and added docs